package.json

```diff
@@ -78,7 +78,7 @@
     "sinon": "^7.1.1"
   },
   "engines": {
-    "node": ">=8.6.0"
+    "node": ">=8.10.0"
```

Comment:
Why update Node.js? Is that pattern requires higher version of Node.js?

Reply:
It's a very minor breaking change, since the v8 LTS refers to v8.16.0 so user should use that version instead.

The "BREAKING CHANGE" commit message (in this PR) is simply for users with Node < 8.10, which after this PR, their hexo wouldn't run and want to look for the cause.

Reply:
I should note that, despite being a breaking change, because this is very minor, I would release as a patch update (e.g. 4.0.1), not a minor.